### PR TITLE
Release/1.0.2

### LINKED ISSUE #93: Test against WordPress 5.9
**Is your enhancement related to a problem? Please describe.**
 Once WordPress 5.9 is released, we'll want to test Block for Apple Maps to see if any incompatibility issues arise.

**Describe the solution you'd like**
- [ ] test Block for Apple Maps on WordPress 5.9
- [ ] open issues for any incompatibilities noted in testing
- [ ] resolve issues identified in testing
- [ ] bump "tested up to" version
- [ ] if code changes needed due to incompatibilities, ship a plugin release, otherwise use "Plugin asset/readme update" action to update "tested up to" version on .org repo

**Designs**
n/a

**Describe alternatives you've considered**
none

**Additional context**
Related: #78, #82.

### LINKED ISSUE #104: Release version 1.0.2
This issue is for tracking changes for the 1.0.2 release.  :tada:  Target release date: **13 December 2020**.

### Pre-release steps
- n/a

### [Release steps](https://github.com/10up/maps-block-apple/blob/develop/CONTRIBUTING.md#release-instructions)

- [x] Branch: Starting from `develop`, cut a release branch named `release/1.0.2` for your changes.
- [x] Version bump: Bump the version number in `maps-block-apple.php`, `readme.txt`, and `package.json` if it does not already reflect the version being released.  Update both the plugin "Version:" property and the plugin `MAPS_BLOCK_APPLE_VERSION` constant in `maps-block-apple.php`.
- [x] Changelog: Add/update the changelog in `CHANGELOG.md` and `readme.txt`.
- [x] Props: update `CREDITS.md` with any new contributors, confirm maintainers are accurate.
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in `.distignore`.
- [x] Readme updates: Make any other readme changes as necessary. `README.md` is geared toward GitHub and `readme.txt` contains WordPress.org-specific content.  The two are slightly different.
- [x] Merge: Make a non-fast-forward merge from your release branch to `develop` (or merge the pull request), then do the same for `develop` into `trunk` (`git checkout trunk && git merge --no-ff develop`). `trunk` contains the latest stable release.
- [x] Push: Push your `trunk` branch to GitHub (e.g. `git push origin trunk`).
- [x] Release: Create a [new release](https://github.com/10up/maps-block-apple/releases/new), naming the tag and the release with the new version number, and targeting the `trunk` branch. Paste the changelog from `CHANGELOG.md` into the body of the release and include a link to the [closed issues on the milestone](https://github.com/10up/maps-block-apple/milestone/4?closed=1).
- [x] SVN: Wait for the [GitHub Action](https://github.com/10up/maps-block-apple/actions) to finish deploying to the WordPress.org repository. If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.
- [x] Check WordPress.org: Ensure that the changes are live on https://wordpress.org/plugins/maps-block-apple/. This may take a few minutes.
- [x] Close the milestone: Edit the [milestone](https://github.com/10up/maps-block-apple/milestone/4) with release date (in the `Due date (optional)` field) and link to GitHub release (in the `Description` field), then close the milestone.
- [x] Punt incomplete items: If any open issues or PRs which were milestoned for `1.0.2` do not make it into the release, update their milestone to `1.1.0`, or `Future Release`.

### PR DESCRIPTION
**Pull request by jeffpaul**
### Description of the Change

Release and version bump to 1.0.2.

### Alternate Designs

none.

### Benefits

Properly updates plugin version number and alerts consumers to `all the wonderful new things`.

### Possible Drawbacks

none identified.

### Verification Process

Manually verified via the GitHub Desktop UI.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Applicable Issues

Resolves #93, #104.

### Changelog Entry

n/a

### CODE REVIEW

**Comment by jeffpaul** [comment]
@Sidsector9 reminder ping on code review

**Review by Sidsector9 — DISMISSED** [review]

**Comment by jeffpaul** [comment]
I'm going to hold off on merging this until we get #99 reviewed/merged and into this release.

**Review by mitogh — COMMENTED** [review]
LGTM :+1:

**Pull request merged**
